include/cetl/variable_length_array.hpp

```diff
@@ -55,44 +55,6 @@ class VariableLengthArrayBase
     ///
     using size_type = std::size_t;
 
-    constexpr VariableLengthArrayBase(
```

Comment:
I had to move this stuff down so it could use some of the functions defined below.